setup.cfg

```diff
@@ -0,0 +1,33 @@
+[metadata]
+name = manm_cs
+version = 0.0.5
+author = Johannes Huegle, Christopher Hagedorn
```

Comment:
would also add the students who contributed to the package Lukas Boehme, Mats Poerschke, Jonas Umland